src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/__tests__/DownloadedPackagesSQLLiteStoreTests.swift

```diff
@@ -20,7 +20,308 @@ import FMDB
 import XCTest
 
 final class DownloadedPackagesSQLLiteStoreTests: XCTestCase {
-	private var store: DownloadedPackagesSQLLiteStore = .inMemory()
+	#if INTEROP
+
+	private var store: DownloadedPackagesSQLLiteStoreV1 = .inMemory()
+
+	override func setUp() {
+		super.setUp()
+		store.close()
+	}
+
+	func testEmptyEmptyDb() throws {
+		store.open()
+		XCTAssertNil(store.package(for: "2020-06-13", country: "DE"))
+	}
+
+	// Add a package, try to get it, assert that it matches what we put inside
+	func testSettingDays() throws {
+		store.open()
+		let keysBin = Data("keys".utf8)
+		let signature = Data("sig".utf8)
+
+		let package = SAPDownloadedPackage(
+			keysBin: keysBin,
+			signature: signature
+		)
+		store.set(country: "DE", day: "2020-06-12", package: package)
+		let packageOut = store.package(for: "2020-06-12", country: "DE")
+		XCTAssertNotNil(packageOut)
+		XCTAssertEqual(packageOut?.signature, signature)
```

Comment:
Thanks for not using plain `XCTAssert()`! 🎉

Reply:
❤️